config/data/dataset/iwcd_cnn_short.yaml

```diff
@@ -1,4 +1,7 @@
 _target_: watchmal.dataset.cnn_mpmt.cnn_mpmt_dataset.CNNmPMTDataset
 mpmt_positions_file: /data/WatChMaL/data/IWCDshort_mPMT_image_positions.npz
-collapse_arrays: False
+mode: ['charge']
+collapse_mode: None
 padding_type: double_cover
+scaling_charge: [2.634658, 6.9462004]
+scaling_time: [1115.6687, 263.4307]
```

Comment:
Is it possible to leave the default configuration without these hard-coded scaling parameters? In my testing so far, this scaling seems to make the performance significantly worse, but I'm continuing to test to see whether that is the problem or not.